packages/devtools-plugin/src/devtools.plugin.ts

```diff
@@ -57,6 +57,11 @@ export class NgxsReduxDevtoolsPlugin implements NgxsPlugin {
 
           this.devtoolsExtension!.send({ ...action, type }, newState);
         }
+      }),
+      catchError(error => {
+        const type = getActionTypeFromInstance(action);
+        this.devtoolsExtension!.send({ ...action, type }, this.store.snapshot());
+        throw error;
```

Comment:
We could also get here if something in the `tap` part throws an error (like an error calling the dev tool).
Maybe we could use the `finally` pipe instead and combine the code?
Also, if there is an error for the `@@INIT` action would the dev tools extension not get initialized?

Reply:
The problem is that Rx doesn't pass anything as arguments to the `finalize` operator, no error and no `nextState`

Reply:
Hmmm. Then could you catch before the `tap`?

Reply:
I'll try